dcos/dcos-ui
# [spike day] DCOS-34504: Align LDAP Authentication radio buttons

Change left margin to align authentication radio buttons

Closes https://jira.mesosphere.com/browse/DCOS-34504

## Testing
1. Go to Settings > LDAP Direcotory
2. Click the "Add Directory" button
3. Open the Authentication tab
4. Verify that the radio buttons are aligned.
5. Verify that this change doesn't break any other positions

## Trade-offs
I think it will be good to add a top margin as well (something like 6px). @lsamimi , what do you think?

## Dependencies
https://github.com/mesosphere/dcos-ui-plugins-private/pull/902

## Screenshots

### Before
![screenshot from 2018-12-21 17-11-24](https://user-images.githubusercontent.com/40791275/50349409-61e7b180-0544-11e9-9518-11750e1f227c.png)

### After
![screenshot from 2018-12-21 17-12-54](https://user-images.githubusercontent.com/40791275/50349418-68762900-0544-11e9-9b01-d4bede92d40b.png)

Edit:
![screenshot from 2019-01-02 10-37-22](https://user-images.githubusercontent.com/40791275/50584903-eeff0480-0e7a-11e9-8aa9-080a293cb809.png)
